am/templates/api/api-service.yaml

```diff
@@ -43,7 +43,7 @@ spec:
       appProtocol: http
       {{ end }}
       {{- end }}
-      name: {{ .Values.api.name }}-technical
+      name: {{ printf "%s-%s" (.Values.api.name | trunc 5 | trimSuffix "-") "technical" }}
```

Comment:
On Service resource, names are limited to 63 chars so here the trunc should be done at 53 chars in order to be able to append "-technical"